modules/src/ics24_host/error.rs

```diff
@@ -1,31 +1,33 @@
-use anomaly::{BoxError, Context};
-use thiserror::Error;
+use displaydoc::Display;
+use std::string::String;
 
-pub type ValidationError = anomaly::Error<ValidationKind>;
+pub type ValidationError = ValidationKind;
 
-#[derive(Clone, Debug, Error, PartialEq, Eq)]
+impl flex_error::StdErr for ValidationKind {}
```

Comment:
This doesn't really make sense.

Reply:
Here because of the use of `displaydoc`.